src/main/java/edu/hm/hafner/grading/Score.java

```diff
@@ -29,17 +29,19 @@ public abstract class Score<S extends Score<S, C>, C extends Configuration> impl
 
     private final String name;
     private final String icon;
+    private final String baseline;
     private final C configuration;
     @SuppressWarnings("serial")
     private final List<S> subScores;
 
     @SafeVarargs
     @SuppressWarnings("varargs")
-    Score(final String name, final String icon, final C configuration, final S... scores) {
+    Score(final String name, final String icon, final String baseline, final C configuration, final S... scores) {
```

Comment:
Here we have the same thing: can't we get rid of baseline by looking into the configuration?